### PR TITLE
Python 2/3 compatibility

### LINKED ISSUE #11: ModuleNotFoundError: No module named 'models'
I get the following error:
```
parsely_slackbot
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python37-32\Scripts\parsely_slackbot-script.py", line 11, in <module>
    load_entry_point('parsely-slackbot==1.1.0', 'console_scripts', 'parsely_slackbot')()
  File "c:\program files (x86)\python37-32\lib\site-packages\pkg_resources\__init__.py", line 487, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "c:\program files (x86)\python37-32\lib\site-packages\pkg_resources\__init__.py", line 2728, in load_entry_point
    return ep.load()
  File "c:\program files (x86)\python37-32\lib\site-packages\pkg_resources\__init__.py", line 2346, in load
    return self.resolve()
  File "c:\program files (x86)\python37-32\lib\site-packages\pkg_resources\__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "c:\program files (x86)\python37-32\lib\site-packages\parsely_slackbot\__init__.py", line 1, in <module>
    from .slackbot import SlackBot
  File "c:\program files (x86)\python37-32\lib\site-packages\parsely_slackbot\slackbot.py", line 8, in <module>
    from parsely import parsely, models
  File "c:\program files (x86)\python37-32\lib\site-packages\parsely\parsely.py", line 1, in <module>
    from models import Post, Author, Section, Topic, Tag, Referrer, Shares
ModuleNotFoundError: No module named 'models'
```

Windows Server 2016, python 3.7 32bit

### PR DESCRIPTION
**Pull request by dan-blanchard**
Should fix Parsely/slackbot#11.

### CODE REVIEW

**Comment by dan-blanchard** [comment]
This was mostly me running `python-modernize` on the codebase + a little manual tweaking to remove unnecessary `list()` wrapping calls that generates. I've run the tests locally and it all seems to still work.

File: parsely/tests.py [hunk 1]
```diff
@@ -72,78 +79,97 @@ def test_shares_detail(self):
...
         def handle(res):
             self.assertTrue(res[7].hits > 0)
+
         self.p.analytics(aspect="authors", _callback=handle)
 
     def test_analytics_one_pubdate(self):
         with self.assertRaises(ValueError):
-            self.p.analytics(aspect="authors", pub_start=datetime(2017, 9, 04))#2013, 10, 01))
+            self.p.analytics(
+                aspect="authors", pub_start=datetime(2017, 9, 0o4)
```
Comment (emmett9001) [inline parsely/tests.py]: This `0o4` is weird, should just be `4`

**Review by emmettbutler — APPROVED** [review]
LGTM with one little error

File: parsely/tests.py (hunk 1 above)
Comment (dan-blanchard) [reply parsely/tests.py]: Good catch!  `04` isn't valid syntax in Python 3. Not sure why the default assumption is octal though.

**Pull request merged**